packages/x-date-pickers-pro/src/MultiInputDateRangeField/MultiInputDateRangeField.tsx

```diff
@@ -29,6 +29,14 @@ const MultiInputDateRangeFieldSeparator = styled(
   },
 )({});
 
+const MultiInputDateRangeFieldTextField = styled(MuiTextField, {
+  name: 'MuiMultiInputDateRangeField',
+  slot: 'TextField',
+  overridesResolver: (props, styles) => styles.textField,
+})({
+  // flexGrow: 1,
```

Comment:
I'm testing using the `fullWidth` prop @LukasTy , I'll remove the comment if we decide to go for this prop :+1:

Reply:
IMHO seems to be working fine. 👌 And will be easily overridden through props if need be. 👍 
The only thing left is to replicate the change in `MultiInputDateTimeRangeField` and `MultiInputTimeRangeField`. 👍